src/web/components/ParticipantManagement/UpdateParticipantDialog.stories.tsx

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable no-console */
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { ParticipantStatus } from '../../../api/entities/Participant';
+import { UserRole } from '../../../api/entities/User';
+import UpdateParticipantDialog from './UpdateParticipantDialog';
+
+const meta: Meta<typeof UpdateParticipantDialog> = {
+  component: UpdateParticipantDialog,
+  title: 'Manage Participants/Update Participant Dialog',
+};
+export default meta;
+
+type Story = StoryObj<typeof UpdateParticipantDialog>;
+
+export const ParticipantWithExistingRoles: Story = {
+  args: {
+    apiRoles: [
+      { id: 1, roleName: 'Role1', externalName: 'Role 1' },
```

Comment:
Perhaps we should use actual role names rather than placeholders? It has caused some confusion in the past.